Terning-iOS/Terning-iOS/Source/Presentation/Home/HomeViewController.swift

```diff
@@ -146,6 +149,9 @@ extension HomeViewController: UICollectionViewDataSource {
             
         case .filtering:
             let cell = rootView.collectionView.dequeueReusableCell(withReuseIdentifier: FilteringCell.className, for: indexPath) as! FilteringCell
+            
+            cell.delegate = self
+            
             return cell
```

Comment:
```suggestion
            let cell = rootView.collectionView.dequeueReusableCell(withReuseIdentifier: FilteringCell.className, for: indexPath) as? FilteringCell ?? UICollectionViewCell()
            
            cell.delegate = self
            
            return cell
```
강제 언래핑 (as!) 대신 안전한 언래핑 (as?)을 사용하는 방식을 더 추천합니다! 실패할 경우 강제 종료 되는 오류를 막을 수 있습니다!

Reply:
닐 코얼레싱 말고!
guard let 바인딩으로 해결해주세요!!!

Reply:
헉... 안그래도 swiftLint에 떠서 고치려고 했는데 빠르게 수정해보겠습니닷